scripts/build-binaries.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o xtrace
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/../hack/utils.sh"
+readonly REPO_PATH=$(get_root_path)
+readonly CAPDCTL_BIN=${REPO_PATH}/bin/capdctl
+readonly CAPDCTL_SRC=${REPO_PATH}/cmd/capdctl
+
+readonly CAPDMGR_BIN=${REPO_PATH}/bin/capd-manager
+readonly CAPDMGR_SRC=${REPO_PATH}/cmd/capd-manager
+
+readonly KIND_TEST_BIN=${REPO_PATH}/bin/kind-test
+readonly KIND_TEST_SRC=${REPO_PATH}/cmd/kind-test
+
+source "${REPO_PATH}/hack/set-workspace-status.sh"
+
+go build -ldflags "-X versioninfo.GitBranch=${GIT_BRANCH} \
```

Comment:
Need to make this w/ less copy pasta.

Reply:
could set the ldflags to a string var and reuse:

```
LDFLAGS="-X ..."

go build -ldflags "${LDFLAGS}" -o ...
```

Reply:
Fixed.